src/Apps/Artist/Routes/Overview/Components/ArtistCareerHighlights.tsx

```diff
@@ -22,6 +22,10 @@ interface ArtistCareerHighlightsProps {
 const ArtistCareerHighlights: FC<ArtistCareerHighlightsProps> = ({
   artist,
 }) => {
+  if (!artist || !artist.insights.length) {
```

Comment:
!artist?.insights?.length

Reply:
if there's an issue with the data fidelity on `insights` we'd get a runtime error trying to guard for the error :)

Reply:
yes!!